lua/godoc/pickers/fzf_lua.lua

```diff
@@ -1,13 +1,31 @@
+local core = require("fzf-lua.core")
+local builtin = require("fzf-lua.previewer.builtin")
```

Comment:
Unfortunately, we can't have these require statements here in the global scope, or users who don't have fzf-lua will experience a crash. You'll have to move this into one of the functions. Have a look at how this is done for e.g. the snacks picker.